WalletWasabi.Tests/UnitTests/Crypto/StrobeTests.cs

```diff
@@ -55,6 +55,10 @@ public void BehaiorIsDeterministic()
 			var strobe3 = strobe1.MakeCopy();
 
 			Assert.Equal(strobe1.ToString(), strobe3.ToString());
+
+			strobe1.AddData(ToBytes("Something"), false);
+			
+			Assert.NotEqual(strobe1.ToString(), strobe3.ToString());
```

Comment:
in addition to ensuring both objects diverge it'd make sense to ensure that `strobe1` is still in the state that it was in.

perhaps just compare the buffer state directly by saving a copy of `strobe1.ToString()` before the next call to `AddAssociatedData`? but see also comment below about `ToString`, i don't think that should be part of the public API, but i'm not sure how to achieve that in the way that the data driven tests code is structured...